ksp_plugin/interface_flight_plan.cpp

```diff
@@ -84,7 +84,7 @@ Burn GetBurn(Plugin const& plugin,
       parameters.primary_index = plugin.CelestialIndexOfBody(
           *barycentric_rotating_reference_frame->primary());
       parameters.secondary_index = plugin.CelestialIndexOfBody(
-          *barycentric_rotating_reference_frame->secondary());
+          *barycentric_rotating_reference_frame->secondaries().front());
```

Comment:
Maybe check that there is only one?  Or, if there is a convention that `front` is special, comment it.